api/controllers/console/app/workflow_run.py

```diff
@@ -1,11 +1,13 @@
 from typing import Literal, cast
```

Comment:
we should also check the service_api and do proper handling.

Reply:
when a service_api request to an archived data?  I will handle it when I test on it

Reply:
Yeah, maybe raise a BadRequest